Prac06/converting.py

```diff
@@ -0,0 +1,22 @@
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.core.window import Window
+
+class miles_to_kilometres(App):
+    def build(self):
+        Window.size = (400, 300)
+        self.title = "Converting Calculator"
+        self.root = Builder.load_file('converting.kv')
+        return self.root
+
+    def handle_calculate(self, value):
+        result = value * 1.60934
+        self.root.ids.output_label.text = str(result)
+
+    def handle_increment(self,increment):
+            self.root.ids.input_number.text = str(int(self.root.ids.input_number.text) + increment)
```

Comment:
It might be useful to add a try catch in here for the integer casting, just to catch any non-integer inputs and to prevent crashes. Something like this:
```
input_number = 0
try:
    input_number = int(self.root.ids.input_number.text)
catch ValueError:
    #input_number remains 0
self.root.ids.input_number.text = str(input_number + increment)
```

Also, you could encapsulate the above error checking into a function that returns `'input_number'`, making it usable in more than one place if needed.